StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp9/MaintainabilityRules/SA1413CSharp9UnitTests.cs

```diff
@@ -3,9 +3,62 @@
 
 namespace StyleCop.Analyzers.Test.CSharp9.MaintainabilityRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.Testing;
     using StyleCop.Analyzers.Test.CSharp8.MaintainabilityRules;
+    using Xunit;
+    using static StyleCop.Analyzers.Test.Verifiers.StyleCopCodeFixVerifier<
+        StyleCop.Analyzers.MaintainabilityRules.SA1413UseTrailingCommasInMultiLineInitializers,
+        StyleCop.Analyzers.MaintainabilityRules.SA1413CodeFixProvider>;
 
     public class SA1413CSharp9UnitTests : SA1413CSharp8UnitTests
     {
+        [Fact]
+        [WorkItem(3240, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3240")]
+        public async Task VerifyWithInitializerAsync()
```

Comment:
💡 Can you add a test showing that no diagnostic is reported for single-line `with` expressions?

Reply:
Updated